cl/users/utils.py

```diff
@@ -272,6 +272,30 @@ def delete_user_assets(user: User) -> None:
         "please see our contact page, https://www.courtlistener.com/contact/",
         "from_email": settings.DEFAULT_FROM_EMAIL,
     },
+    "not_confirmed_edu_account": {
+        "subject": "Confirm Your .edu Membership Email",
+        "body": "Hello, %s,\n\n"
+        "Thank you for requesting a .edu membership.\n\n"
+        "We noticed that your email address has not yet been confirmed in our "
+        "system. To complete your membership request, please confirm your email "
+        "address by following the link we sent when you registered.\n\n"
+        "------------------\n\n"
+        "Need help? Please visit our contact page: https://www.courtlistener.com/contact/",
```

Comment:
We can remove the name here to simplify, but I think we should provide the confirmation link so they don't have to dig around in their email:

```suggestion
        "body": "Hello,\n\n"
        "Thank you for requesting a .edu membership.\n\n"
        "We noticed that your email address has not yet been confirmed in "
        "CourtListener.com. To complete your membership request, please "
        "confirm your email address by clicking the following link:
        "  XXX\n\n"
        "------------------\n\n"
        "Need help? Please visit our contact page: https://www.courtlistener.com/contact/",
```